### ISSUE TITLE
Support web apps and PWAs

### ISSUE THREAD

**Issue by ralyodio** [socketsupply/socket#125]
I don't see web app listed. It's the one thing missing.

**Comment by chicoxyzzy** [comment]
What do you mean by web apps and PWA support? We do not prohibit the use of any Web APIs, they should just work.

**Comment by heapwolf** [comment]
Im not sure I understand what this issue is asking for, can you rephrase your question with a bit more detail?

**Comment by ralyodio** [comment]
It says its for desktop and mobile. what about web? running and serving from a domain?

**Comment by chicoxyzzy** [comment]
Socket Runtime provides additional capabilities like Bluetooth and UDP sockets, which are not available on web platform. That means that it's possible to wrap web applications into Socket Runtime, but not other way around if the Socket app uses these capabilities. Does that answer your question?

**Comment by ralyodio** [comment]
No. I don't follow. Sorry I just heard about this project. It seems there's another project for desktop/mobile and they don't do web either

**Comment by heapwolf** [comment]
It sounds like you're looking for a web browser.
